packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx

```diff
@@ -146,7 +146,7 @@ export const PaymentConfirmation = ({
             <a
               data-testid="download-link"
               className="button download-link"
-              href={productUrl}
+              href={productUrl + `/?email=${encodeURIComponent(email)}`}
```

Comment:
nit: I'd suggest not mixing concatenation and template strings /driveby

Reply:
moved to its own variable outside of template